lib/ancestry/has_ancestry.rb

```diff
@@ -4,16 +4,23 @@ def has_ancestry options = {}
       # Check options
       raise Ancestry::AncestryException.new("Options for has_ancestry must be in a hash.") unless options.is_a? Hash
       options.each do |key, value|
-        unless [:ancestry_column, :orphan_strategy, :cache_depth, :depth_cache_column, :touch].include? key
+        unless %i[ancestry_column orphan_strategy cache_depth depth_cache_column touch ancestry_id_column ancestry_uniqueness].include? key
           raise Ancestry::AncestryException.new("Unknown option for has_ancestry: #{key.inspect} => #{value.inspect}.")
         end
       end
 
-
       # Create ancestry column accessor and set to option or default
       cattr_accessor :ancestry_column
       self.ancestry_column = options[:ancestry_column] || :ancestry
 
+      # Create ancestry id column accessor and set to option or default
+      cattr_accessor :ancestry_id_column
+      self.ancestry_id_column = options[:ancestry_id_column] || :id
```

Comment:
why `:id` here?

Reply:
`:id` would be the default.  maybe `#primary_key` would be better choice